Marlin/src/pins/stm32f1/pins_MKS_ROBIN_PRO.h

```diff
@@ -29,6 +29,8 @@
   #error "Oops! Select an STM32F1 board in 'Tools > Board.'"
 #elif HOTENDS > 3 || E_STEPPERS > 3
   #error "MKS Robin pro supports up to 3 hotends / E-steppers. Comment out this line to continue."
+#elif STM32_HAS_USB_SERIAL
+  #error "A serial port is set to -1, but this board does not support native USB. Set SERIAL_PORT to 1 for the onboard USB port."
```

Comment:
I just realized that I copied and pasted the same sentence everywhere, but SERIAL_PORT 1 is not always correct. I know it is supposed to be 3 for this board. Others are probably slightly more random as well.

Reply:
This board also failed tests, because when I tested locally I was pointed to my own branch of example files...